homeassistant/components/sensibo/sensor.py

```diff
@@ -178,6 +178,7 @@ class SensiboDeviceSensorEntityDescription(SensorEntityDescription):
         value_fn=lambda data: data.co2,
         extra_fn=None,
     ),
+    *DEVICE_SENSOR_TYPES,
```

Comment:
For the fuutre, it would be nice to have a test to assert expected entities by device type as well.

Reply:
Yes. snapshot came to light after all these tests were already in place. Should be updated.